toqito/state_props/common_epistemic_overlap.py

```diff
@@ -0,0 +1,91 @@
+"""Calculate the common epistemic overlap of quantum states."""
+
+from itertools import product
+
+import numpy as np
+
+from toqito.matrix_ops.vec import vec
+
+
+def common_epistemic_overlap(states, dim=None) -> float:
```

Comment:
Is it necessary to include the dimension as input? Is this not a property of the `states` that are provided to this function?